hayha/cloudformation_types.py

```diff
@@ -6,10 +6,13 @@
 from .dataflow import Resource, SecurityResource
 
 class CloudFormationResource:
-    def __init__(self, direct_flow=None, reverse_flow=None, managed_flow=None,
-            direct_protection_flow=None, reverse_protection_flow=None,
-            managed_protection_flow=None,
-            direct_protected_from_flow=None, reverse_protected_from_flow=None,
+    #  Here are some name changes I thought made things easier to understand.
+    #  I've only changed the names here. Let me know what you think, so I can
+    #  change the names accross the file (or not).
+    def __init__(self, outgoing_connection=None, incoming_connection=None, connection_to_form=None,
+            outgoing_entrance_protection=None, incoming_entrance_protection=None,
+            connection_to_protect=None,
+            outgoing_exit_protection=None, incoming_exit_protection=None,
```

Comment:
`outgoing_connection` OK
`incoming_connection` OK
`connection_to_form` -> typo?
`*_entrance_protection` -> doesn't make sense to me
`connection_to_protect` OK
`*_exit_protection` -> doesn't make sense to me

I like `entrance` and `exit`, but I don't like the `incoming` and `outgoing` parts, that is confusing and doesn't seem to describe well what's going on. Maybe `entrance_protection_of` and `entrance_protection_by` (and similar for `exit`)?

When you change your PR, please modify your current commit (with `git commit --amend`), and `push --force` to your branch. That will make the commit history a bit cleaner :)